src/openrct2/interface/Window_internal.h

```diff
@@ -14,6 +14,8 @@
 #include <list>
 #include <memory>
 
+#include <src/openrct2/windows/tile_inspector.h>
+
```

Comment:
```suggestion
enum class TileInspectorPage : int16_t;

```
Can you try forward declaring it, instead? If it works we avoid one include

Reply:
:+1: